ui/app/AppLayouts/Chat/CommunityComponents/CommunityMembersPopup.qml

```diff
@@ -0,0 +1,175 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+import "../../../../imports"
+import "../../../../shared"
+import "../../../../shared/status"
+import "./"
+import "../components"
+
+ModalPopup {
+    id: popup
+    property QtObject community: chatsModel.activeCommunity 
+
+    header: Item {
+        height: childrenRect.height
+        width: parent.width
+    
+        StyledText {
+            id: groupName
+            text: qsTr("Members")
+            anchors.top: parent.top
+            anchors.topMargin: 2
+            anchors.left: parent.left
+            font.bold: true
+            font.pixelSize: 14
+            wrapMode: Text.WordWrap
+        }
+
+        StyledText {
+            text: community.nbMembers.toString()
+            width: 160
+            anchors.left: parent.left
+            anchors.top: groupName.bottom
+            anchors.topMargin: 2
+            font.pixelSize: 14
+            color: Style.current.darkGrey
+        }
+    }
+
+    CommunityPopupButton {
+        id: inviteBtn
+        label: qsTr("Invite People")
+        width: popup.width
+        iconName: "invite"
+        onClicked: openPopup(inviteFriendsPopup)
+        Component {
+            id: inviteFriendsPopup
+            InviteFriendsToCommunityPopup {
+                onClosed: {
+                    destroy()
+                }
+            }
+        }
+    }
+
+    Separator {
+        id: sep
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.top: inviteBtn.bottom
+        anchors.topMargin: Style.current.smallPadding
+        anchors.leftMargin: -Style.current.padding
+        anchors.rightMargin: -Style.current.padding
+    }
+
+    ListView {
+        id: memberList
+        anchors.top: sep.bottom
+        anchors.topMargin: Style.current.smallPadding
+        anchors.bottom: popup.bottom
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.bottomMargin: Style.current.halfPadding
+        spacing: 4
+        Layout.fillWidth: true
+        Layout.fillHeight: true
+        model: community.members
+        delegate: Item {
+            id: contactRow
+            width: parent.width
+            height: identicon.height
+
+            property string nickname: {
+                // Get contact nickname
+                const contactList = profileModel.contacts.list
+                const contactCount = contactList.rowCount()
+                let nickname = ""
+                for (let i = 0; i < contactCount; i++) {
+                    if (contactList.rowData(i, 'pubKey') === model.pubKey) {
+                        return contactList.rowData(i, 'localNickname')
```

Comment:
Do we do that search for nickname a lot? If so, we might want to put that in a function that reused